gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/user/register/RegisterConfirmationEndpoint.java

```diff
@@ -111,7 +110,8 @@ public void handle(RoutingContext routingContext) {
                 );
     }
 
-    private String getTemplateFileName(Client client) {
-        return "registration_confirmation" + (client != null ? FormManager.TEMPLATE_NAME_SEPARATOR + client.getId() : "");
+    @Override
+    public String getTemplateSuffix() {
+        return Template.REGISTRATION_CONFIRMATION.template();
```

Comment:
Template.REGISTRATION_CONFIRMATION.template() is used instead of "registration_confirmation". 

Are you sure you want to remove everything after?
 (client != null ? FormManager.TEMPLATE_NAME_SEPARATOR + client.getId() : "")

Reply:
ok, that's fine